apps/src/gamelab/animationListModule.js

```diff
@@ -646,9 +650,19 @@ function loadAnimationFromSource(key, callback) {
  * Action creator for adding an animation.
  * @param {!AnimationKey} key
  * @param {SerializedAnimation} props
+ * @param {number} [index]
  * @returns {{type: string, key: AnimationKey, props: SerializedAnimation}}
  */
-export function addAnimationAction(key, props) {
+export function addAnimationAction(key, props, index) {
+  // Spritelab projects add animation at the beginning of animation list
+  if (index >= 0) {
```

Comment:
Why >=, not ==?

Reply:
My thought with >= was that it allows us to extend this feature in the future - in case there's ever a case where we want to add animations at another index. So I chose >= so that I could check if the index is present. Happy to lock-it down though - I don't imagine much of a future case where we'd add animations other than at the front or back of the list.

Reply:
Hmm I think == is clearer for now then, and we could always change it in the future if desired

Reply:
Note: If you do decide not to go with >=, use ===, not ==. I don't think you would run into issues with undefined==0, but future refactors might cause issues.